
<launch>
    <param name="max-depth-demand"    value="3" />                    <!-- Maximum depth demand (m) -->
    <param name="over-depth"          value="8" />                    <!-- Maximum depth (m) - causes abort -->
    <param name="over-pitch"          value="45" />                   <!-- Maximum pitch (deg) - causes abort -->
    <param name="over-roll"           value="90" />                   <!-- Maximum roll (deg) - causes abort -->
    <param name="max-internal-temp"   value="45" />                   <!-- Maximum internal pressure vessel temp (deg) - causes abort -->
    <param name="min-motor-voltage"   value="19000" />                <!-- Minimum Motor voltage (mV) - causes abort -->
    <param name="mission-timeout"     value="30" />                   <!-- Mission timeout (minutes) -->
    <param name="lat_orig"            value="50.936545" />            <!-- Origin Latitude -->
    <param name="long_orig"           value="-1.405103" />            <!-- Origin Longitude -->
    <param name="ImportWaypoints"     value="False" />                <!-- Import Waypoints -->
    <param name="look_ahead_distance" value="10" />                   <!-- look ahead distance (in m) for track following algorithm  effectivly inverse of gain-->
    
    <node name="mission_script" pkg="delphin2_mission" type="mission_script_kantapon_testEastleigh.py"  output="screen" required="true" /> 
    
    <!-- Eastleigh North Pier (50.957024,-1.366769) -->
    <!-- Eastleigh South Pier (50.956473,-1.366835) -->
        
    <!-- Add all other nodes under here! -->
    
    <node name="auv_sim" pkg="delphin2_mission" type="auv_sim.py" required="true" output="screen"/>

    <node name="logger" pkg="hardware_interfaces" type="logger.py" required="true" output="screen"/>

</launch>
